pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -69,74 +94,92 @@ def map_fips(self, fips):
         fips: str
             County FIPS code to map.
         """
-        logging.info(f"Mapping output to WebUI for {self.state}, {fips}")
+        log.info("Mapping output to WebUI for %s, %s", self.state, fips)
```

Comment:
To the best of my knowledge f"" string formatting is the modern Python way and % formatting is out of style.
For logging I much prefer a completely static message string and extra={} for variables or to be even more cutting edge try
```
log = structlog.get_logger()
log.info("Mapping output to WebUI", state=self.state, fips=fips)
```

Reply:
oO! I just saw that you started layering in structured logging. I'll convert over to that. 

Regarding the f-strings vs % formatting, I think logging is still the exception -- but I'm sure the preferred style still varies: https://docs.python.org/3/howto/logging.html#optimization

Reply:
Hmm, I can't actually get this to work with structured logging. I tried, but keep getting this error: 
```
Traceback (most recent call last):
  File "/Users/flyrrecovery/.pyenv/versions/covid-data-model/bin/pyseir", line 11, in <module>
    load_entry_point('pyseir', 'console_scripts', 'pyseir')()
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/envs/covid-data-model/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/envs/covid-data-model/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/envs/covid-data-model/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/envs/covid-data-model/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/envs/covid-data-model/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/flyrrecovery/src/github/covid-data-model/pyseir/cli.py", line 435, in build_all
Revert "Switches webui adaptor to structured logging."
    states_only=states_only,
  File "/Users/flyrrecovery/src/github/covid-data-model/pyseir/cli.py", line 211, in _build_all_for_states
    p.map(states_only_func, states)
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/lib/python3.7/multiprocessing/pool.py", line 268, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/Users/flyrrecovery/.pyenv/versions/3.7.7/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
TypeError: msg() got an unexpected keyword argument 'state'
```